dotcom-rendering/src/components/SlotBodyEnd/ReaderRevenueEpic.tsx

```diff
@@ -173,11 +173,12 @@ export const ReaderRevenueEpic = ({
 			'contributions-epic-module',
 		);
 
-		window
-			.guardianPolyfilledImport(module.url)
-			.then((epicModule: { ContributionsEpic: EpicType }) => {
+		import(
+			/* webpackChunkName: "contributions-epic" */ `../marketing/epics/ContributionsEpic`
+		)
+			.then((epicModule) => {
 				endPerformanceMeasure();
-				setEpic(() => epicModule.ContributionsEpic); // useState requires functions to be wrapped
+				setEpic(() => epicModule.ContributionsEpic);
```

Comment:
I think this is a worthwile comment to keep, it’s quite confusing that if you pass a function, [React assumes you want it to be an **updater**](https://react.dev/reference/react/useState#is-using-an-updater-always-preferred)